tests/page-layout-both/OPS/package.opf

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<package xmlns="http://www.idpf.org/2007/opf" xmlns:epub="http://www.idpf.org/2007/ops" version="3.0" xml:lang="en" unique-identifier="pub-id">
+  <metadata xmlns:dc="http://purl.org/dc/elements/1.1/">
+    <dc:coverage>Fixed Layout</dc:coverage>
+    <dc:creator>Wendy Reid</dc:creator>
+    <dc:date>2021-11-24</dc:date>
+    <dc:description>If a reflowable spine item follows a pre-paginated one, it should appear on the next page.</dc:description>
+    <dc:identifier id="pub-id">page-layout-both</dc:identifier>
+    <dc:language>en</dc:language>
+    <dc:title>page-layout-both</dc:title>
+    <link rel="dcterms:rights" href="https://www.w3.org/Consortium/Legal/2015/copyright-software-and-document"/>
+    <link rel="dcterms:rightsHolder" href="https://www.w3.org"/>
+    <meta property="dcterms:isReferencedBy">https://www.w3.org/TR/epub-33/#property-layout-local</meta>
+    <meta property="dcterms:isReferencedBy">https://www.w3.org/TR/epub-rs-33/#page-layout-both</meta>
+    <meta property="dcterms:modified">2021-01-01T00:00:00Z</meta>
+    <meta property="rendition:layout">pre-paginated</meta>
```

Comment:
The date on line 15 is wrong (but there's already a different comment thread on that line).